algorithm.go

```diff
@@ -2,6 +2,8 @@ package cose
 
 import (
 	"crypto"
+	_ "crypto/sha256"
+	_ "crypto/sha512"
```

Comment:
Can you document those import in the package documentation since it might not be desired?

Reply:
You mean removing the imports, adding them to `conformance_test.go` so tests can pass and documenting this requirement?